docs/docs/tailwind-css.md

```diff
@@ -94,34 +92,24 @@ module.exports = {
 }
 ```
 
-**Option B**: Install stable `tailwind.macro` and use Tailwind 0.7.4
-
-```bash
-// Remove tailwind 1.0.0+ if you've already installed it
-npm uninstall tailwindcss
-
-// Install tailwind 0.7.4 and stable tailwind.macro
-npm install tailwindcss@0.7.4
-npm install tailwind.macro
-```
-
-2. Use the Babel Macro (`tailwind.macro`) in your styled component
+4. Use twin.macro to create your styled component
 
 ```javascript
```

Comment:
Users could be a little confused about where this goes, so we typically advise showing a complete component. Might as well update it while we're in there.
````suggestion
```jsx:title=src/pages/index.js
````